liiweb/templates/liiweb/legislation_list.html

```diff
@@ -16,11 +16,18 @@ <h1 class="my-4">{% trans 'Legislation' %}</h1>
           {% include "peachjam/_help_button.html" %}
         {% endif %}
       {% endblock %}
-      {% if show_unconstitutional_provisions %}
+      {% if show_more_resources %}
         <h5>{% trans "More resources" %}</h5>
-        <div class="mb-3">
-          <a href="{% url "unconstitutional_provision_list" %}">{% trans "Unconstitutional provisions" %}</a>
-        </div>
+        {% if show_unconstitutional_provisions %}
```

Comment:
to avoid weird padding between them, use

```html
<ul class="list-unstyled">
<li>...</li>
<li>...</li>
</ul>
```

Bootstrap's list-unstyled prevents any bullets to the side, but is useful for a list like this.